src/main/resources/api/service_openapi.yaml

```diff
@@ -428,8 +428,8 @@ components:
           type: string
           nullable: true
         reference:
-          description: The reference to the data (i.e. gs://...) #todo- describe as json instead
-          type: object
+          description: The reference to the data (i.e. gs://...)
```

Comment:
how about something like "serialized object containing details specific to the referenceType. For instance, TDR snapshots must be serialized json."